pydax/_schema.py

```diff
@@ -72,7 +82,16 @@ def export_schema(self, *keys: str) -> SchemaDict:
 
     @property
     def retrieved_url_or_path(self) -> Union[_typing.PathLike, str]:
-        "The URL or path from which the schema was retrieved."
+        """The URL or path from which the schema was retrieved.
+
+        :return: The URL or path from which the schema was retrieved.
```

Comment:
`@property` methods are documented as attributes: https://pydax.readthedocs.io/en/latest/api-references/pydax.schema.Schema.html#pydax.schema.Schema

Shouldn't need this `:return:` line.